.github/workflows/reusable-markdown-link-check.yml

```diff
@@ -22,5 +22,6 @@ jobs:
             --exclude "^http://code.google.com/p/concurrentlinkedhashmap$"
             --exclude "^https://softwareengineering.stackexchange.com/questions/29727"
             --max-retries 6
-            --max-concurrency 1
+            --max-concurrency 4
```

Comment:
when I initially aded the concurrency limit higher concurrency limits still got rate limited, hope that it works now

Reply:
yeah, I'm also getting different results now than previously, I suspect they tweaked the rate limit again